webapp/src/pages/EditUserPage/EditUserPage.css

```diff
@@ -0,0 +1,58 @@
+.edit-user-page {
+  margin: 0;
+  padding: var(--padding-top-bottom) var(--padding-sides);
+  height: 100vh;
+  background-color: var(--color-background);
+  color: var(--color-light);
+  display: grid;
+  gap: 4rem var(--column-gap);
+  grid-template-columns: 1fr;
+  grid-template-rows: auto auto 1fr auto;
+  overflow: scroll;
+}
+
+.edit-user-form {
+  min-width: 15rem;
+  max-width: 20rem;
+  justify-self: center;
+}
+
+.edit-user-form-button {
+  justify-self: center;
+  align-self: end;
+}
+
+@media all and (min-width: 768px) {
+  .edit-user-page {
+    display: grid;
+    grid-template-columns: repeat(var(--column-number), 1fr);
+    grid-template-rows: auto 1fr auto;
+    grid-row-gap: 8rem;
```

Comment:
I used this ` gap: 4rem var(--column-gap);` in the user page in the beginning because I thought it was convenient, but the semantics of the gap (that you cannot configure independently on a particular per-gap basis) do not mesh well with the one-time nature of this particular gap. Also, as it is now, it is off sync with the users page. I would match it with what you have there:
Five rows in total, the second one replaces this gap, the fourth one replaces your second gap.
(template should be something like     `grid-template-rows: auto 20% auto 1fr auto`). Then you'd have to fix the respective rows of the actual classes you have in the selectors below...

Reply:
The result is the same but the code is more consistent. Done